test/Bridges/Variable/flip_sign.jl

```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
         @test MOI.get(bridged_mock, MOI.VariablePrimalStart(), y) == 1
     end
 
+    @testset "Test Mock model" begin
+        var_names = ["a", "b", "c", "d"]
+        MOI.set(mock, MOI.VariableName(), MOI.get(mock, MOI.ListOfVariableIndices()), var_names)
+
+        con_d = MOI.get(mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorOfVariables, MathOptInterface.Zeros}())[1]
+        con_bc = MOI.get(mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorOfVariables, MathOptInterface.Nonnegatives}())
+        con_ex = MOI.get(mock, MOI.ListOfConstraintIndices{MathOptInterface.VectorAffineFunction{Float64}, MathOptInterface.Zeros}())[1]
```

Comment:
Same than for variable names, you should not set the name here of the constraint that are not bridged. It should be set only once for both `mock` and `bridged_mock`.